docs/narr/viewconfig.rst

```diff
@@ -1025,6 +1029,106 @@ and the browser representation of the error will include the same information.
 See :ref:`environment_chapter` for more information about how, and where to set
 these values.
 
+.. index::
+   single: Accept
+   single: Accept content negotiation
+
+.. _accept_content_negotiation:
+
+Accept Header Content Negotiation
+---------------------------------
+
+The ``accept`` argument to :meth:`pyramid.config.Configurator.add_view` can be used to control :term:`view lookup` by dispatching to different views based on the HTTP ``Accept`` request header.
+Consider the example below in which there are three defined views.
+Each view uses the ``Accept`` header to trigger an appropriate response renderer.
+
+.. code-block:: python
+
+    from pyramid.httpexceptions import HTTPNotAcceptable
+    from pyramid.view import view_config
+
+    @view_config(accept='application/json', renderer='json')
+    @view_config(accept='text/html', renderer='templates/hello.jinja2')
+    def myview(request):
+        return {
+            'name': request.GET.get('name', 'bob'),
+        }
+
+    @view_config()
+    def myview_unacceptable(request):
+        raise HTTPNotAcceptable
+
+The appropriate view is selected here when the client specifies an unambiguous header such as ``Accept: text/*`` or ``Accept: application/json``.
+Similarly, if the client specifies a media type that no view is registered to handle, such as ``Accept: text/plain``, it will fall through to ``myview_unacceptable`` and raise ``406 Not Acceptable``.
+There are a few cases in which the client may specify ambiguous constraints:
```

Comment:
In L1061 and L1063, the header is not ambiguous or unambiguous, it's just that a header could be specified such that more than one view matches? (So the ambiguity is in the view matching, rather than in the header itself.) Might it be worth rephrasing for people who are less familiar with Accept headers?

Reply:
I just pushed some changes that I think make this more clear.